ci/qa/copyright-correction.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Get all PHP files in src/ directory
+files=$(find src/ -type f -name "*.php")
+
+for file in $files
+do
+    # Get the year of the first commit of the file
+    year=$(git log --follow --format=%ad --date=format:'%Y' $file | tail -1)
+
+    # Replace the year in the copyright statement
+    sed -i "s/Copyright [0-9]\{4\} SURFnet bv/Copyright $year SURFnet bv/g" $file
+done
```

Comment:
Nice little script. I don't think this belongs in the ci/qa folder tho. The bin folder is a better place for these kind of scripts.

Reply:
Allright, moved it